Test.C

```diff
@@ -581,3 +549,65 @@ void encryptionStore(struct Player *playerPtr, struct Encryption encryptionStock
+    int equipSellSelect;
+    int equipSellAmount;
+    int equipSellTransaction;
+    char equipSellAnswer;
+
+    printf("\n");
+    printf("|----------------------------- Equipment Selling ------------------------------|\n");
+    printf("Equipment that you own:\n");
+    printf(" # | Equipment Name                       | Cost  | Message Amount | Inventory |\n");
+    printf("---|--------------------------------------|-------|----------------|-----------|\n");
+    for(int i=0; i<NUM_EQUIPMENT; i++){
+        printf(" %d | %36s | %5d | %14d | %9d |\n", i + 1, equipmentStock[i].name, equipmentStock[i].cost, equipmentStock[i].messageAmount, playerPtr->equipmentInventory[i]);
+    }
+
+    printf("Your current credits are: %d\n", playerPtr->currentCredits); // current credit
+        printf("Please select the equipment you would like to sell or select %d to pass: ", NUM_EQUIPMENT + 1); // chose 11 because there are 10 options rn
+        scanf("%d", &equipSellSelect);
+
+    while (equipSellSelect < NUM_EQUIPMENT){ // while the user is selecting something to sell and not choseing to pass
+        printf("You have selected: %s\n", equipmentStock[0].name); // not sure if that should be equipmentStock or not
+        printf("How many would you like to sell?\n");
+        scanf("%d", &equipSellAmount);
+        equipSellTransaction = equipmentStock[0].cost * equipSellAmount; // not sure if that should be equipmentStock or not
+        printf("This will sell for: %d\n", equipSellTransaction);
+        printf("Are you sure you want to sell? y/N ");
+        scanf(" %c", &equipSellAnswer);
+            if (equipSellAnswer == 'y' || equipSellAnswer == 'Y'){ 
+                if (playerPtr->equipmentInventory[0] > 0){ // checks that the user has the item in thier inventory
+                    printf("Thank You!\n");
+                    playerPtr->currentCredits = playerPtr->currentCredits + equipSellTransaction;
+                    printf("Your current credits are: %d\n", playerPtr->currentCredits); // current credit
+                    playerPtr->equipmentInventory[0] = playerPtr->equipmentInventory[0] - equipSellAmount; 
+                }
+                if (playerPtr->equipmentInventory == 0){ // checks that the user has the item in their inventory
+                    printf("You do not have any of this equipment in your inventory\n");
+                    printf("The transaction was canceled\n");
+                }
+            }
+            if (equipSellAnswer == 'n' || equipSellAnswer == 'N'){
+                printf("The transaction was canceled\n");
+            }
+            // print equipment that is owned
+            printf("Equipment that you own:\n");
+            printf(" # | Equipment Name                       | Cost  | Message Amount | Inventory |\n");
+            printf("---|--------------------------------------|-------|----------------|-----------|\n");
+            for(int i=0; i<NUM_EQUIPMENT; i++){
```

Comment:
Space necessary here per the style guide
```suggestion
            for (int i=0; i<NUM_EQUIPMENT; i++){
```